client/views/users/templates.html

```diff
@@ -185,12 +178,15 @@ <h3 class="bottom" style="margin-right: 40px;">
     </a>
 </template>
 
-<template name="avatar">
-    <div class="member {{# if draggable }}js-member{{else}}js-member-on-card-menu{{/ if}}">
-        <span class="member-initials" title="{{ user.profile.name }} ({{ user.username }})">
-            {{ firstChar user.profile.name }}
-        </span>
-        <span class="member-status {{# if user.profile.status}}active{{/if}}"></span>
-        <span class="member-type {{ memberType }}"></span>
+<template name="userAvatar">
+    <div class="member {{size}} {{ class }} {{# if draggable }}js-member{{else}}js-member-on-card-menu{{/ if}}"
+         title="{{ userData.profile.name}} ({{ userData.username}})">
+        {{> avatar user=userData size=size }}
+        {{#if showStatus }}
+            <span class="member-status {{# if userData.profile.status}}active{{/if}}"></span>
+        {{/if }}
+        {{#if showMemberType}}
```

Comment:
It seems that every time you call `{{> userAvatar }}` with the `showStatus` option, you also use the `showMemberType`. Maybe we could have only one option that would display the two icons. (:+1: for the option BTW)